definitions/subworkflows/molecular_alignment.cwl

```diff
@@ -85,6 +96,19 @@ steps:
             description: sample_name
        out:
             [duplex_seq_metrics]
+    grouped_bam_to_cram:
+        run: ../tools/bam_to_cram.cwl
+        in:
+            bam: grouped_clip_overlap/grouped_clipped_bam
+            reference: reference
+        out:
+            [grouped_cram]
```

Comment:
It's complaining about this line.  The name of the output from the bam_to_cram tool is not `grouped_cram` but `cram`.
https://github.com/genome/analysis-workflows/blob/64e10813d527c67ed7987697d5c47cdbbaadbebd/definitions/tools/bam_to_cram.cwl#L26-L27